### PR TITLE
Update driver

### PR DESCRIPTION
**Pull request by michalmuskala**

### CODE REVIEW

**Comment by michalmuskala** [comment]
It needs some clean-up, and changes to the driver to bring back logging, but it works :smiley:

File: test/test_helper.exs [hunk 1]
```diff
@@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
 Application.put_env(:ecto, TestRepo,
                     adapter: Mongo.Ecto,
                     url: "ecto://localhost:27017/ecto_test",
-                    pool_size: 1)
+                    size: 1)
```
Comment (josevalim) [inline test/test_helper.exs]: @ericmj Ecto has recently changed this flag to `pool_size`. Can we do the same for `mongo` for consistency?

Comment (ericmj) [inline test/test_helper.exs]: The option is up to each pool adapter. Are the driver's adapters used or are we creating new ones?

Comment (michalmuskala) [inline test/test_helper.exs]: I'm using the driver's adapters. Ecto simply translates `pool_size` to `size` before handing options to poolboy.

Comment (ericmj) [inline test/test_helper.exs]: Okay, we can do the same in the driver.

On Wed, Jul 29, 2015 at 11:13 AM, Michał Muskała <notifications@github.com>
wrote:

> In test/test_helper.exs
> <https://github.com/michalmuskala/mongodb_ecto/pull/35#discussion_r35740758>
> :
>
> > @@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
> >  Application.put_env(:ecto, TestRepo,
> >                      adapter: Mongo.Ecto,
> >                      url: "ecto://localhost:27017/ecto_test",
> > -                    pool_size: 1)
> > +                    size: 1)
>
> I'm using the driver's adapters. Ecto simply translates pool_size to size
> before handing options to poolboy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/michalmuskala/mongodb_ecto/pull/35/files#r35740758>.
>



-- 
Eric Meadows-Jönsson

**Comment by michalmuskala** [comment]
Done! :smile:

**Pull request merged**